nx-dev/data-access-ai/src/lib/data-access-ai.ts

```diff
@@ -54,6 +60,14 @@ export async function nxDevDataAccessAi(
   sources: { heading: string; url: string }[];
   sourcesMarkdown: string;
 }> {
+  if (!supabaseClient) {
+    console.log('Creating supabase client');
```

Comment:
This is probably not supposed to be here.

Reply:
Fixed!